house/house.js

```diff
@@ -1,22 +1,43 @@
 const fs = require('fs')
 const xml2js = require('xml2js')
+const moment = require('moment')
+const request = require('request')
 
 BillFetcher.prototype.FetchBills = function (date, cb) {
   if (typeof date == "function") {
     cb = date
+    if (this.File && this.File != "") {
```

Comment:
nit: `if (this.File)` will already ensure `this.File` is not the empty string as `""==False` in JS